sessions/slides/introduction-to-nowcasting.qmd

```diff
@@ -56,19 +56,22 @@ Remember from [biases in delay estimation](../biases-in-delay-distributions):
 - reporting of events can be triggered by the **secondary** event
 - in that case, longer delays might be missing because whilst the *primary events* have occurred the *secondary events* **have not occurred yet**
 
-Nowcasting is exactly this if the seondary event is the data being available for analysis.
+Nowcasting is exactly this!.
 
-### Nowcasting as a population-level problem
+### A simple approach to nowcasting
 
-If we don't have individual-level data, we can estimate the delay distribution of reporting from multiple snapshots
-
-![](figures/germany_historical.png)
+1. Estimate the delay distribution from external data
```

Comment:
```suggestion
1. Estimate the delay distribution from other data
```
Could be the same, e.g. the individual-level data using the approach in biases in delay distributions (my assumption was that that is the suggestion here - would anyone really ever use other data sets for this?